src/BUILD.gn

```diff
@@ -54,6 +54,7 @@ if (chip_build_tests) {
       "${chip_root}/src/messaging/tests",
       "${chip_root}/src/protocols/bdx/tests",
       "${chip_root}/src/protocols/secure_channel/tests",
+      "${chip_root}/src/protocols/secure_channel/tests",
```

Comment:
This change is not needed (as the previous line is exactly the same). Likely, due to a recent rebase..

Reply:
Oops